macros/contexts/contextBoolean.pl

```diff
@@ -0,0 +1,160 @@
+## contextBoolean.pl
+
+sub _contextBoolean_init { context::Boolean::Init() }
+
+package context::Boolean;
+
+sub Init {
+	my $context = $main::context{Boolean} = Parser::Context->getCopy("Numeric");
+	$context->{name} = "Boolean";
+	$context->{value}{Formula} = "context::Boolean::Formula";
+
+	## Disable unwanted operations and functions
+	$context->functions->disable('All');
+	#    $context->parens->undefine('<');
+	$context->operators->undefine('-', '/', ' /', '/ ', '//', '^', '**', 'u+', '!');
+	$context->operators->redefine('><');
+
+	## Re-enable necessary operations
+	$context->operators->set(
+		'+'   => { class => "Parser::BOP::or",  string => ' or ',  TeX => '\vee ',   perl => '||' },
+		'or'  => { class => "Parser::BOP::or",  string => ' or ',  TeX => '\vee ',   perl => '||' },
+		'and' => { class => "Parser::BOP::and", string => ' and ', TeX => '\wedge ', perl => '&&' },
+		'*'   => { class => "Parser::BOP::and", string => ' and ', TeX => '\wedge ', perl => '&&' },
+		' *'  => { class => "Parser::BOP::and", string => ' and ', TeX => '\wedge ', perl => '&&' },
+		'* '  => { class => "Parser::BOP::and", string => ' and ', TeX => '\wedge ', perl => '&&' },
+		# implicit multiplication must have string '*' for Parser::Op (L#314)
+		' '   => { class => "Parser::BOP::and", string => ' *',    TeX => '\wedge ' },
+		'u-'  => { class => "Parser::UOP::not", string => '-',     TeX => '\mathord{\sim}' },
+		'><'  => { class => "Parser::BOP::xor", string => ' xor ', TeX => '\oplus ' },
+		'xor' => { class => "Parser::BOP::xor", string => ' xor ', TeX => '\oplus ' },
```

Comment:
You also need to add `precedence` values to these so that `and` has higher precedence than `or`.  That will be true for `+` and `-`, but not for your new operators, which have a default precedence that is the same for all of them.  Plus and minus are currently at precedence 1, so you should be able to use 1 for `or'` and 2 for `and`.  I'm not sure where `xor` fits in.

I might recommend getting rid of all of the standard operators and setting them with completely fresh values, rather than relying on existing properties.  The only ones you need that you aren't redefining are `,`,   `.`, and `fn`.  So deleting all the ones you are using and adding them back in might be best.

I also agree with Alex that using `alias` to tie duplicate names to one standard version of each operator will make maintenance easier.

Reply:
Agreed, I'm about to push some changes in this direction.